mcs/models/add_group_request.go

```diff
@@ -24,12 +24,13 @@ package models
 
 import (
 	"github.com/go-openapi/errors"
-	strfmt "github.com/go-openapi/strfmt"
+	"github.com/go-openapi/strfmt"
 	"github.com/go-openapi/swag"
 	"github.com/go-openapi/validate"
 )
 
 // AddGroupRequest add group request
+//
```

Comment:
why did this change? did your swagger make this? or IntelliJ? just wondering.

Reply:
swagger did it